chain-core/src/common/mod.rs

```diff
@@ -27,17 +27,25 @@ pub fn hash256<D: Digest>(data: &[u8]) -> H256 {
 /// Seconds since UNIX epoch
 pub type Timespec = u64;
 
+/// 32-byte for keys or hashes etc.
 pub type H256 = [u8; HASH_SIZE_256];
+/// 33-byte for pubkeys etc.
 pub type H264 = [u8; HASH_SIZE_256 + 1];
+/// 64-byte for sigs etc.
 pub type H512 = [u8; HASH_SIZE_256 * 2];
 
 /// Types of tendermint events created during `deliver_tx` / `end_block`
 #[derive(Debug, Clone, Copy)]
 pub enum TendermintEventType {
+    /// if transaction is valid
     ValidTransactions,
+    /// filter for view pub keys
     BlockFilter,
+    /// validators that were jailed
     JailValidators,
+    /// validators that were slashed -- TODO: it'll be the same? needed?
```

Comment:
@calvinaco  Do you think we need this?

Reply:
If we don't record in here, the information is lost.